week1_intro/primer/recap_pytorch.ipynb

```diff
@@ -336,7 +339,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If you compute gradient from multiple losses, the gradients will add up at variables, therefore it's useful to __zero the gradients__ between iteratons."
+    "If you compute gradient from multiple losses, the gradients will be added to variables, therefore, it's useful to __zero the gradients__ between iteratons."
```

Comment:
The meaning is different. The gradient isn't added to variables; rather, each variable has an attached field called `.grad` which is where the gradients accumulate.